ts/main.ts

```diff
@@ -1 +1,36 @@
 /* global data */
+interface Obj {
+  title: string;
+  photoURL: string;
+  notes: string;
+  entryId: number;
+}
+
+const $inputURL = document.getElementById('photoURL') as HTMLInputElement;
+
+$inputURL?.addEventListener('input', () => {
+  const $img = document.querySelector('img') as HTMLElement;
```

Comment:
Move this to the global space. By including it here, you are going to search the DOM for this element every single time a user presses a key on the keyboard